index.html

```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
+                                <ul class="block2-tag1">
+                                    <li>html</li>
+                                </ul>
+                                <ul class="block2-tag2">
+                                    <li>bootstrap</li>
+                                </ul>
+                                <ul class="block2-tag3">
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="button-container"><button type="submit" class="action">See Project</button></div>
+                        </div>
+                    </div>
+                </div>
+                <div class="second-frame">
+                    <div class="card-works1">
+                        <div class="right-block2">
+                            <h2>Profesional Art Printing Data</h2>
+                            <p>A daily selection of privately personalized reads;
+                               no accounts or sign-ups required. has been the industry's
+                               standard</p>
+                            <div class="block2-tags">
+                                <ul class="block2-tag1">
+                                    <li>html</li>
+                                </ul>
+                                <ul class="block2-tag2">
+                                    <li>bootstrap</li>
+                                </ul>
+                                <ul class="block2-tag3">
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="button-container"><button type="submit" class="action">See Project</button></div>
+                        </div>
+                    </div>
+                </div>
+                <div class="second-frame">
+                    <div class="card-works1">
+                        <div class="right-block2">
+                            <h2>Profesional Art Printing Data</h2>
+                            <p>A daily selection of privately personalized reads;
+                               no accounts or sign-ups required. has been the industry's
+                               standard</p>
+                            <div class="block2-tags">
+                                <ul class="block2-tag1">
+                                    <li>html</li>
+                                </ul>
+                                <ul class="block2-tag2">
+                                    <li>bootstrap</li>
+                                </ul>
+                                <ul class="block2-tag3">
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                            <div class="button-container"><button type="submit" class="action">See Project</button></div>
+                        </div>
```

Comment:
- [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one